packages/component/src/Styles/StyleSet/SendStatus.js

```diff
@@ -14,7 +14,6 @@ export default function createSendStatusStyle({
       backgroundColor: 'transparent',
       border: 0,
       color: accent,
-      cursor: 'pointer',
```

Comment:
Need to look at this one. I believe send status is a `<button>` appear to be a link. So a visually-looking link need a pointer.

The reason it is a `<button>` is that, it is an action, people cannot middle-click to open it, and cannot copy link.

Reply:
![image](https://user-images.githubusercontent.com/14900841/54647909-8b3fad00-4a61-11e9-8e87-e9d85e4ba191.png)
Reverting, because you're right of course :P